cluster/config-defaults.yaml

```diff
@@ -107,7 +107,13 @@ skipper_redis_write_timeout: "25ms"
 
 skipper_cluster_ratelimit_max_group_shards: 1
 
+#
 # skipper routesrv settings
+#
+# skipper_routesrv_enabled is a three state switch:
+# - "false" - routesrv deployment is removed, skipper uses own k8s dataclient
+# - "pre" - routesrv is deployed, skipper uses own k8s dataclient
+# - "exec" - routesrv is deployed, skipper uses routesrv
```

Comment:
Used `exec` instead of `true` similar to `nlb_switch` and to hint that its a three state config

Reply:
no that's not needed here. skipper-ingress waits for first successful route load and routesrv has the time to startup before it hurts skipper-ingress.